COMPATIBILITY.md

```diff
@@ -1,13 +1,17 @@
-# Plugin Compatibility with Workflow
+# Workflow Compatibility Plugins
```

Comment:
Fixed how? There was no subsequent change here.

BTW if there is a line comment in a GH PR, and you subsequently add a commit changing that line (or one nearby), GH will automatically hide the thread from the _Files changed_ tab (and show the comment as being _on an outdated diff_ in the _Conversation_ tab). That is the usual way of tracking which comments are outstanding.